gpu-plugins/overlays/amd/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../base
+- daemonset-amdgpu-device-plugin-daemonset-kube-system.yaml
```

Comment:
We have daemonset in the file name twice, maybe remove one of them unless there's a formatting reason?

Reply:
I know it's a bit weird but it's following the convention we decided on: `<resource-kind>-<resource-name>-<namespace>.yaml`. In this case, `resource-name` has `daemonset` in it.

Reply:
Gotcha. Cool, as long as there's a reason for it.

Reply:
To reduce the number of "-", I've been combining the pieces for my naming.
Example: "daemonset-amdgpudeviceplugindaemonset-kubesystem.yaml"